openupgrade_scripts/scripts/sale_management/18.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Le Filament (https://le-filament.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def _fill_sale_order_template_sequence(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+         ALTER TABLE sale_order_template ADD COLUMN IF NOT EXISTS sequence INTEGER
+         """,
+    )
```

Comment:
LGTM, but FYI: we use now `openupgrade.add_columns(env, [("sale.order.template", "sequence", "integer")])`